File: src/Entity/Panier.php
<?php

namespace App\Entity;

use App\Repository\PanierRepository;
use Doctrine\ORM\Mapping as ORM;

#[ORM\Table(name: "panier", indexes: [
    new ORM\Index(name: "idUser", columns: ["idUser"])
])]
#[ORM\Entity(repositoryClass: PanierRepository::class)]
class Panier
{
    #[ORM\Column(name: "idPanier", type: "integer", nullable: false)]
    #[ORM\Id]
    #[ORM\GeneratedValue(strategy: "IDENTITY")]
    private $idpanier;

    #[ORM\ManyToOne(targetEntity: Utilisateur::class)]
    #[ORM\JoinColumn(name: "idUser", referencedColumnName: "idUser")]
    private $iduser;

    public function getIdpanier(): ?int
    {
        return $this->idpanier;
    }

    public function getIduser(): ?Utilisateur
    {
        return $this->iduser;
    }

    public function setIduser(?Utilisateur $iduser): static
    {
        $this->iduser = $iduser;

        return $this;
    }
}

File: src/main/java/entities/Panier.java
package entities;

import service.ProduitService;

public class Panier {
    int idPanier;
    Utilisateur utilisateur;

    public Panier()
    {

    }
    public Panier(int idPanier,Utilisateur utilisateur)
    {
        this.idPanier=idPanier;
        this.utilisateur=utilisateur;
    }

    public int getIdPanier() {
        return idPanier;
    }


    public Utilisateur getUtilisateur() {
        return utilisateur;
    }

    public void setIdPanier(int idPanier) {
        this.idPanier = idPanier;
    }
}
